### DISCUSSION TITLE
Unable to Switch GitHub Classroom Repository to Public

### DISCUSSION THREAD

**Discussion by FCLMKJ** [community/community#133077] [GitHub Education] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hello,

Previously, I was able to switch a GitHub Classroom repository from private to public, but now it seems impossible.

The radio checkbox is disabled, and I am receiving the following message:
"You cannot change the visibility of an assignment after students have accepted it."

Could you please provide a solution for this issue?

Thank you.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @FCLMKJ ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by RyanHecht** [comment, marked as answer]
Hi @FCLMKJ,

You can no longer change the visibility of an assignment after students have accepted it. You can change the visibility of the assignment repositories manually on GitHub.com, or you can create a new assignment with "Public" visibility and have students re-accept.
